### PR TITLE
{lib}[intel/2018b] phonopy v1.14.2 w/ Python 2.7.15

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PyYAML/PyYAML-3.13-intel-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['3ef3092145e9b70e3ddd2c7ad59bdd0252a94dfe3949721633e41344de00a6bf']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('libyaml', '0.2.1'),
+]
+
```
Comment (vanzod) [inline easybuild/easyconfigs/p/PyYAML/PyYAML-3.13-intel-2018b-Python-2.7.15.eb]: Missing `download_dep_fail = True` and `use_pip = True`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar04.gastly.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 2.7.5
See https://gist.github.com/612dc6be0772c26bca31ceed995e8e90 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/354c530748a0e2b9c2df659f67022322 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c0cfc5285d553d56258f1e3d1dae5957 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node2420.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1f45f1f96388a72114352b6a619d4032 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/4fb0f5d1c29f788b2e39a535864268e8 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**